elements/stacinfo/src/main.ts

```diff
@@ -47,6 +47,9 @@ export class EOxStacInfo extends LitElement {
   @property({ type: Array })
   footer: Array<string> = [];
 
+  @property()
```

Comment:
For completeness, please add `@property({ attribute: "style-override" })` so it also can be set via HTML attribute `style-override` in a vanilla use case!